docs/book/features/block_volume.md

```diff
@@ -0,0 +1,278 @@
+# Volume Provisioning
+
+There are two types of volume provisioning in a kubernetes cluster:
+
+1. Dynamic Volume Provisioning
+2. Static Volume Provisioning
+
+## Dynamic Volume Provisioning
```

Comment:
Can we put a table somewhere in this section and list down what storage class is supported for block volume on each flavor of driver?